mslib/static/templates/user/reset_confirmation.html

```diff
@@ -0,0 +1,7 @@
+<p>Reset your password?</p>
+<br>
+<p>If you requested a password reset for {{username}}, click below button to complete the process.</p>
+<a href="{{reset_password_url}}"><button>Reset Password</button></a>
+<p> If you didn't make this request, ignore this email</p>
+<br>
+<p>Cheers!</p>
```

Comment:
better a new line at  the end. 

less possible conflicts on different filesystems and OS